### ISSUE TITLE
Research groups can aggregate their collections on a dedicated landing page

### ISSUE THREAD

**Issue by brianraymor** [chanzuckerberg/single-cell#312]
### Context

_To increase awareness and impact, research groups want to aggregate their submitted collections on a dedicated landing page with custom branding opportunities. This may also provide a substitute for external sites that reference visualizations in cellxgene Explorer such as [Tabula Sapiens](https://tabula-sapiens-portal.ds.czbiohub.org/)._

_Data consumers want to easily find and explore all datasets submitted by a specific research group._

### Existing Designs

* [Broad SCP Study Collections](https://singlecell.broadinstitute.org/single_cell/collections) and [documentation](https://singlecell.zendesk.com/hc/en-us/articles/4412832822683-Collections-Overview) 
* [USSC Cell Browser - Project Filtering](https://cells.ucsc.edu/?proj=Allen+Brain+Atlas&ds=allen-celltypes).
* [Jackson Laboratory (JAX) publication pages](https://singlecell.jax.org/datasets/vph-2020) demonstrate a summary image or graphical abstract.

### Notes

1. A common request is for a branding logo. 
1. A potential first step is to add a _my research group_ dropdown to Collections. To ensure a standard set of labels, this list could be "curated" by Lattice. See [**wrangling**](https://czi-sci.slack.com/archives/C024HCSH9PT/p1654548754861629). This collection metadata would be the source for a new filter in _filter by metadata_.

**Comment by brianraymor** [comment]
@hthomas-czi @niknak33 - closing this issue as low priority. We did address the addition of a consortia filter if not special landing pages.
